codegen/projections/high_score_service/lib/high_score_service/types.rb

```diff
@@ -172,17 +171,6 @@ module Types
       include Hearth::Structure
     end
 
-    # @!attribute errors
```

Comment:
Types module should still be alphabetical if possible.

Reply:
Fixed